### PR TITLE
feat: [CDS-41144]: added cypress test case for azure web app 

### PR DESCRIPTION
**Pull request by vikrant-harness**
### Summary

added cypress test case for azure web app 

#### Screenshots

https://user-images.githubusercontent.com/106532291/181226043-6fc415c2-908d-4f3d-8f07-7a91616ce5af.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by vikrant-harness** [comment]
retrigger titlecheck

**Comment by vikrant-harness** [comment]
retrigger titlecheck

**Comment by vikrant-harness** [comment]
trigger cypress

File: cypress/integration/70-pipeline/AzureWebApp.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    cy.contains('span', connectorRef).should('be.visible')
+    cy.contains('span', subscriptionId).should('be.visible')
+  }
+  it('end to end testing for azure web app', () => {
+    cy.visit(newPipelineRoute, { timeout: 30000 })
+    cy.visitPageAssertion()
+
+    // creating a new pipeline
+    cy.get('input[name="name"]').should('be.visible').type('test-pipeline').should('have.value', 'test-pipeline')
+    cy.get('.StyledProps--padding-top-xlarge > .Button--variation-primary > .bp3-button-text').click()
```
Comment (PratyushGar) [inline cypress/integration/70-pipeline/AzureWebApp.spec.ts:49]: We can target the element or text as style is subject to change and so does the class here.

File: cypress/integration/70-pipeline/AzureWebApp.spec.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    cy.get('@runtimeValue').click()
+    cy.wait(500)
+    cy.get('@multiSelectButtons').eq(2).click()
+    cy.get('a.bp3-menu-item').should('have.length', 3).as('valueList')
+    cy.get('@valueList').eq(0).should('contain.text', 'Fixed value').as('fixedValue')
+    cy.get('@valueList').eq(1).should('contain.text', 'Runtime input').as('runtimeValue')
+    cy.get('@valueList').eq(2).should('contain.text', 'Expression').as('expressionValue')
+    cy.get('@runtimeValue').click()
+    cy.wait(500)
+    cy.get('@multiSelectButtons').eq(3).click()
```
Comment (PratyushGar) [inline cypress/integration/70-pipeline/AzureWebApp.spec.ts:117]: We can move `cy.get('@multiSelectButtons').eq(1/2/3)` to one block using `.each` and extract inside assertions outside to reduce code duplication.

**Review by PratyushGar — APPROVED** [review]

File: cypress/integration/70-pipeline/AzureWebApp.spec.ts (hunk 1 above)
Comment (vikrant-harness) [reply cypress/integration/70-pipeline/AzureWebApp.spec.ts]: changed in the latest commit @PratyushGar

File: cypress/integration/70-pipeline/AzureWebApp.spec.ts (hunk 2 above)
Comment (vikrant-harness) [reply cypress/integration/70-pipeline/AzureWebApp.spec.ts]: changed in the latest commit @PratyushGar

**Comment by vikrant-harness** [comment]
trigger cypress

File: cypress/support/75-cd/constants.ts [hunk 3]
```diff
@@ -12,6 +12,9 @@ export const createEnvironmentGroupsCall = `/ng/api/environmentGroup?accountIden
 export const environmentGroupDetailsCall = `/ng/api/environmentGroup/testEnvGroup?routingId=${accountIdentifier}&accountIdentifier=${accountIdentifier}&orgIdentifier=${orgIdentifier}&projectIdentifier=${projectIdentifier}`
 export const environmentGroupYamlSchemaCall = `/ng/api/yaml-schema?routingId=${accountIdentifier}&entityType=EnvironmentGroup&projectIdentifier=${projectIdentifier}&orgIdentifier=${orgIdentifier}&accountIdentifier=${accountIdentifier}&scope=project`
 export const environmentConfigurationCall = `/ng/api/environmentsV2/${environmentIdentifier}?routingId=${accountIdentifier}&accountIdentifier=${accountIdentifier}&orgIdentifier=${orgIdentifier}&projectIdentifier=${projectIdentifier}`
+export const environmentFetchCall = `/ng/api/environmentsV2?routingId=${accountIdentifier}&accountIdentifier=${accountIdentifier}&orgIdentifier=${orgIdentifier}&projectIdentifier=${projectIdentifier}`
+export const environmentSaveCall = `/ng/api/environmentsV2?routingId=${accountIdentifier}&accountIdentifier=${accountIdentifier}`
+
```
Comment (devinder-harness) [inline cypress/support/75-cd/constants.ts:17]: urls look same `&orgIdentifier=${orgIdentifier}&projectIdentifier=${projectIdentifier}` is the only diff

**Review by devinder-harness — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

**Comment by vikrant-harness** [comment]
trigger cypress

**Comment by vikrant-harness** [comment]
retrigger titlecheck

**Pull request merged**